docs/user-guide/scheduled-jobs.md

```diff
@@ -0,0 +1,155 @@
+be deleted. To clean up those jobs and pods, you need to list all jobs created by the scheduled job, and delete them all:
+
+```shell
+$ kubectl get jobs
+NAME            DESIRED   SUCCESSFUL   AGE
+pi-1063102732   1         1            7m
+pi-1133619459   1         1            4m
+...
+
+$ kubectl delete jobs pi-1063102732 pi-1133619459 ...
+job "pi-1063102732" deleted
+job "pi-1133619459" deleted
+...
+```
+
+Once the jobs are deleted, the pods created by them are deleted as well. Note that all jobs created by scheduled
+job "pi" will be prefixed "pi-". You can delete them at once with `kubectl delete jobs --all`, if you want to 
+delete all jobs in the system (not just the ones created by "pi".)
+
+## Scheduled Job Limitations
+
+A scheduled job creates a job object _about_ once per execution time of its schedule. We say "about" because there
+are certain circumstances where two jobs might be created, or no job might be created. We attempt to make these rare,
+but do not completely prevent them. Therefore, jobs should be _idempotent_.
+
+The job is responsible for retrying pods, parallelism among pods it creates, and determining the success or failure
+of the set of pods. A scheduled job does not examine pods at all.
+
+## Writing a Scheduled Job Spec
+
+As with all other Kubernetes configs, a scheduled job needs `apiVersion`, `kind`, and `metadata` fields. For general
+information about working with config files, see [deploying applications](/docs/user-guide/deploying-applications), 
+[configuring containers](/docs/user-guide/configuring-containers), and
+[using kubectl to manage resources](/docs/user-guide/working-with-resources) documents.
+
+A scheduled job also needs a [`.spec` section](https://github.com/kubernetes/kubernetes/tree/{{page.githubbranch}}/docs/devel/api-conventions.md#spec-and-status).
+
+### Schedule 
+
+The `.spec.schedule` is a required field of the `.spec`. It takes a [Cron](https://en.wikipedia.org/wiki/Cron) format
+string, e.g. `0 * * * *` or `@hourly`, as schedule time of its jobs to be created and executed. 
+
+### Job Template
+
+The `.spec.jobTemplate` is another required field of the `.spec`. It is a job template. It has exactly the same schema
+as a [Job](/docs/user-guide/jobs), except it is nested and does not have an `apiVersion` or `kind`, see
+[Writing a Job Spec](/docs/user-guide/jobs/#writing-a-job-spec).
+
+### Starting Deadline Seconds
+
+The `.spec.startingDeadlineSeconds` field is optional. It stands for the deadline (in seconds) for starting the job
+if it misses its scheduled time for any reason. Missed jobs executions will be counted as failed ones. If not specified, 
+there's no deadline.  
+
+### Concurrency Policy
+
+The `.spec.concurrencyPolicy` field is also optional. It specifies how to treat concurrent executions of a job.
+Only one of the following concurrent policies may be specified:
+
+* `Allow` (default): allows concurrently running jobs 
```

Comment:
... created by the same scheduledJob. (If there are multiple scheduledJobs, their respective jobs are always allowed to run concurrently)